pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -110,7 +110,7 @@ type GlobalRoleBinding struct {
 // +kubebuilder:resource:scope=Cluster
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// RoleTemplate holds configuration for a template that is used to create roles and clusterRoles for a cluster or project.
+// RoleTemplate holds configuration for a template that is used to create kubernetes native roles and clusterRoles for a cluster or project.
```

Comment:
```suggestion
// RoleTemplate holds configuration for a template that is used to create native Kubernetes roles and cluster roles for a cluster or project.
```

Reply:
I'm not super open to changing this text any further. This was what was agreed on in thread, and exists to correct an issue already open for this.

Outside of that `clusterRoles` is closer to the proper nomenclature for this than `cluster roles` since this is a k8s resource.

Reply:
The K8s docs authors themselves use regular words often when they mention the resources themselves and not Kinds.
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings

Reply:
Those are the user facing docs. If you do `kubectl explain ClusterRole`, you'll note that it uses Capitalized Camel Case, which is what we are trying to mirror here.